src/app/components/albums-list/albums-list.component.ts

```diff
@@ -5,42 +5,43 @@ import { SpotifyApiService } from '../../services/spotify.service';
 @Component({
   selector: 'app-albums-list',
   templateUrl: './albums-list.component.html',
-  styleUrls: ['./albums-list.component.scss']
+  styleUrls: ['./albums-list.component.scss'],
 })
 export class AlbumsListComponent implements OnInit, OnDestroy {
   public albums: any[] = [];
   public subscription$: Subscription;
-  public noResultsMessage: string = '';
+  public noResultsMessage = '';
   public albumsTotal: number;
   public pageInfo: any;
 
-  constructor(private spotifyService: SpotifyApiService) { }
+  constructor(private spotifyService: SpotifyApiService) {}
 
   ngOnInit() {
     this.getAlbums();
     this.updateList();
   }
 
   getAlbums() {
-    this.spotifyService.getAlbums()
-      .subscribe(
-        ({ albums }: any) => {
-          const { items } = albums;
-          this.albums = items;
-        },
-        (error: any) => console.log(error));
+    this.spotifyService.getAlbums().subscribe(
+      ({ albums }: any) => {
+        const { items } = albums;
+        this.albums = items;
+      },
+      (error: any) => console.log(error)
+    );
   }
 
   updateList() {
     this.noResultsMessage = '';
-    this.subscription$ = this.spotifyService.dataList$
-      .subscribe((data: any) => {
+    this.subscription$ = this.spotifyService.dataList$.subscribe(
+      (data: any) => {
         const { items, total, limit } = data;
         if (!items.length) this.noResultsMessage = 'No results found';
```

Comment:
is this operation is necessary?
You have this part in your template:
```
    <h4 *ngIf="!albums.length">
        {{noResultsMessage}}
    </h4>
```
You can simply remove that action from `updateList()` method and add this message directly into template:
```
    <h4 *ngIf="!albums.length">No results found</h4>
```